components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -861,13 +863,16 @@ def start_webui(instance_id: str, clp_config: CLPConfig, mounts: CLPDockerMounts
     container_webui_logs_dir = pathlib.Path("/") / "var" / "log" / component_name
 
     # Read and update settings.json
+    database_config = Database.parse_obj(clp_config.database)
```

Comment:
Why do we need the `Database.parse_obj`?

Reply:
because we need `database_config.get_clp_connection_params_and_type(True)` to get the dictionary that contains the `table_prefix`. The prefix is not directly exposed in clp_config.
See https://github.com/y-scope/clp/blob/329eb7f4857b1c231d6850d791b4984de092d705/components/clp-py-utils/clp_py_utils/clp_config.py#L161

Reply:
But isn't `clp_config.database` already a `Database` object?